media/js/base/mozilla-fxa-form.js

```diff
@@ -110,7 +155,16 @@ if (typeof window.Mozilla === 'undefined') {
         var contextField = formElem.querySelector('[name="context"]');
         var serviceField = formElem.querySelector('[name="service"]');
         var userVer = parseFloat(Mozilla.Client._getFirefoxVersion());
+        var self = this;
+        var useUITourForFxA = userVer > 79 && Mozilla.UITour;
```

Comment:
Should this now be 80? Also suggest making the check for `UITour` a little more well formed?

```suggestion
        var useUITourForFxA = userVer > 80 && typeof Mozilla.UITour !== 'undefined';
```

Reply:
Sorry, this should be >= 80